UIKitHomework/UIKitHomework/Model/User.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  User.swift
+//  UIKitHomework
+//
+//  Created by Vlad on 02.10.2022.
+//
+
+import Foundation
+
+/// Структура пользователя:
+/// - login - логин, формат текст, должен быть уникальным
+/// - name - имя, формат текст
+/// - passwor - пароль, формат текст
+/// - users - база данных всех зарегистрированных пользователей
+struct User {
+
+    // MARK: - Register statuses enumiration
+    enum RegisterStatuses {
+        case success
+        case loginNonUnique
+        case passwordsNonEqual
+    }
+
+    // MARK: - Static properties
+    private static let users: [User] = [User(login: "admin", name: "admin", password: "123456")]
+
+    // MARK: - Private properties
+    private let login: String
+    private let name: String
+    private let password: String
+
+    // MARK: - Init
+    init(login: String, name: String, password: String) {
+        self.login = login
+        self.name = name
+        self.password = password
+    }
+
+    // MARK: - Public Methods
+    public static func authUser(login: String, password: String) -> Bool {
```

Comment:
```suggestion
    static func authUser(login: String, password: String) -> Bool {
```
public  не используем